src/ResourceManager/ResourceGraph/Commands.ResourceGraph/Az.ResourceGraph.psd1

```diff
@@ -51,7 +51,7 @@
         # ProcessorArchitecture = ''
 
         # Modules that must be imported into the global environment prior to importing this module
-        RequiredModules = @(@{ModuleName = 'Az.Profile'; ModuleVersion = '0.2.0'; })
+        RequiredModules = @(@{ModuleName = 'Az.Profile'; ModuleVersion = '0.2.2'; })
```

Comment:
@cormacpayne The alias for Search-AzureRmGraph comes with 0.2.2, but if this change gets to the gallery before Cloud Shell updates Az.Profile, installing our package will break Cloud Shell again. Please advise which version would be better here.

Reply:
@demyanenko this change is fine

Reply:
@cormacpayne With 0.2.2 I'm not able to run it in current Cloud Shell without updating Az.Profile. Can we leave it at 0.2.0 and update after the shell has caught up? Or are there other ways to approach it?